cusub_perception/localizer/include/localizer/localizer.h

```diff
@@ -32,23 +32,26 @@ namespace localizer_ns
     ros::Publisher pub;
     ros::Subscriber sub;
     int detection_num;
+    void cuprint(std::string str);
+    void cuprint_warn(std::string str);
```

Comment:
Also see last comment. If it is too complicated for C++ to have an imported method, it's not a huge deal

Reply:
Yeah I've tried to share headers between roscpp packages, and I have never been successful. Completely agree it's the way to go though